File: backend/src/main/java/com/ll/hotel/domain/review/comment/repository/ReviewCommentRepository.java
package com.ll.hotel.domain.review.comment.repository;

import com.ll.hotel.domain.review.comment.entity.ReviewComment;
import org.springframework.data.jpa.repository.JpaRepository;
import org.springframework.stereotype.Repository;

@Repository
public interface ReviewCommentRepository extends JpaRepository<ReviewComment, Long> {
}

File: backend/src/main/java/com/ll/hotel/domain/review/comment/repository/ReviewCommentRepository.kt
package com.ll.hotel.domain.review.comment.repository;

import com.ll.hotel.domain.review.comment.entity.ReviewComment;
import org.springframework.data.jpa.repository.JpaRepository;
import org.springframework.stereotype.Repository;

@Repository
interface ReviewCommentRepository : JpaRepository<ReviewComment, Long>
